lang/csharp/src/apache/test/AvroGen/AvroGenHelper.cs

```diff
@@ -93,20 +93,21 @@ public static Assembly CompileCSharpFilesIntoLibrary(IEnumerable<string> sourceF
                 {
                     typeof(object).Assembly.Location,
                     typeof(Schema).Assembly.Location,
+                    typeof(System.CodeDom.Compiler.GeneratedCodeAttribute).Assembly.Location,
                     Path.Combine(assemblyPath, "System.Runtime.dll"),
                     Path.Combine(assemblyPath, "netstandard.dll")
                 };
 
                 // Create compiler
                 CSharpCompilation compilation = CSharpCompilation
-                    .Create(assemblyName)
-                    .WithOptions(new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary))
-                    .AddReferences(assemblies.Select(path => MetadataReference.CreateFromFile(path)))
-                    .AddSyntaxTrees(sourceFiles.Select(sourceFile =>
-                    {
-                        string sourceText = System.IO.File.ReadAllText(sourceFile);
-                        return CSharpSyntaxTree.ParseText(sourceText);
-                    }));
+                .Create(assemblyName)
```

Comment:
Which style defines the tabulation of the chained methods? I am not a fan of tabulating this way.

Reply:
simplified declaration

Reply:
I meant reverting the tab removal :)

Reply:
I chose to make it more legible.

Reply:
Also this formatting is what is default in Visual studio, which we can potentially override in the editorconfig

Reply:
Hmmm, my VS does not do that. Since it is not related to this PR, could you revert that formatting change?

Reply:
Sorry, I've been busy.  Easy fix.  For some reason I thought we were talking about tabbing somewhere else.  Yeah, VS won't change that, won't revert it back either.